resources/lib/modules/updates.py

```diff
@@ -388,7 +388,9 @@ def get_channels(self):
         log.log(str(self.update_json), log.DEBUG)
         if self.update_json:
             for channel in self.update_json:
-                channels.append(channel)
+                # filter versions older than current
+                if float(oe.VERSION_ID) <= float(channel.split('-')[1]):
```

Comment:
Should we check before, if channel contains a `-`? Maybe there are LE forks or update channels without a dash in channel name?

Reply:
I've made it more paranoid. IndexError raises when there's no hyphen as field [1] doesn't exist. If there is a hyphen, it tests that the result is a number. If it comes across something it doesn't understand, it adds it to the list of choices for the user.